core/src/scene/drawRule.h

```diff
@@ -13,17 +13,18 @@
 
 using Color = CSSColorParser::Color;
 using Extrusion = std::pair<float, float>;
+using Offset = std::pair<float, float>;
 
 namespace Tangram {
 
 enum class StyleParamKey : uint8_t {
     none, order, extrude, color, width, cap, join, outline_color, outline_width, outline_cap, outline_join,
     font_family, font_weight, font_style, font_size, font_fill, font_stroke, font_stroke_color, font_stroke_width, font_uppercase,
-    visible, priority
+    visible, priority, offset
 };
 
 struct StyleParam {
-    using Value = variant<none_type, std::string, CapTypes, JoinTypes, Extrusion, int32_t, uint32_t, float, bool>;
+    using Value = variant<none_type, std::string, CapTypes, JoinTypes, Extrusion, Offset, int32_t, uint32_t, float, bool>;
```

Comment:
Might make sense to combine the `Extrusion` and `Offset` variations into a single `std::pair<float, float>` variation, to reduce the generated code size a bit.